src/common/gemm.cpp

```diff
@@ -73,6 +73,7 @@ std::string get_descriptor(dim_t M, dim_t N, dim_t K) {
         status = __VA_ARGS__; \
         double duration_ms = get_msec() - start_ms; \
         std::stringstream ss; \
+        ss.imbue(std::locale::classic()); \
         ss << "cpu,gemm_api,,undef,"; \
```

Comment:
Could we add some a small regression tests suite to ensure the oneDNN functionality works as expected under different locales?